templates/extensions/javascript/package.lint.json

```diff
@@ -3,8 +3,8 @@
   "description": "Greet the world",
   "main": "index.js",
   "dependencies": {
-    "firebase-admin": "^12.1.0",
-    "firebase-functions": "^5.0.0"
+    "firebase-admin": "^12.6.0",
+    "firebase-functions": "^6.0.1"
```

Comment:
@leoortizz Upgraded the extensions as well since last time an upgrade like this was done, it was requested the extensions to be done as well, see:
https://github.com/firebase/firebase-tools/pull/7084#pullrequestreview-2034242334

Can be removed if you think it's better!

Reply:
This is not related to web frameworks, this is for ext:dev:init. We should keep these changes to `templates/extensions`